examples/05_transformers.ipynb

```diff
@@ -0,0 +1,851 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# constructing the transformer\n",
+    "from sktime.transformations.series.boxcox import BoxCoxTransformer\n",
+    "\n",
+    "my_boxcox_trafo = BoxCoxTransformer(method=\"mle\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we apply the constructed transformer `my_trafo` to a (single, univariate) series. First, the transformer is fitted:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# fitting the transformer\n",
+    "my_boxcox_trafo.fit(X_series)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, the transformer is applied, this results in a transformed series."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# transforming the series\n",
+    "my_boxcox_trafo.transform(X_series)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Generally, the series passed to `transform` need not be the same as in `fit` - if it is, the shorthand `fit_transform` can instead be used:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_boxcox_trafo.fit_transform(X_series)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The transformer can also be applied to `Panel` data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_boxcox_trafo.fit_transform(X_panel)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Example: transforming series to primitive features\n",
+    "\n",
+    "The summary transformer can be used to extract sample statistics such as mean and variance from a series. First, we construct the transformer:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# constructing the transformer\n",
+    "from sktime.transformations.series.summarize import SummaryTransformer\n",
+    "\n",
+    "my_summary_trafo = SummaryTransformer()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As before, we can fit/apply with `fit`, `transform`, and `fit_transform`.\n",
+    "\n",
+    "`SummaryTransformer` returns primitive features, hence the output will be a `pandas.DataFrame`, each row corresponding to one series in the input. \n",
+    "\n",
+    "If the input is a single series, the output of `transform` and `fit_transform` will be a one-row `DataFrame`, corresponding to mean etc of that one series:"
```

Comment:
```suggestion
    "If the input is a single series, the output of `transform` and `fit_transform` will be a one-row `DataFrame`, corresponding to the summary statistics of that one series:"
```

Reply:
clarified this, but differently, since not everyone may be able to visualize what "summary statistics" means here